packages/headless/src/features/question-answering/question-answering-common.ts

```diff
@@ -0,0 +1,23 @@
+import {RecordValue, StringValue} from '@coveo/bueno';
```

Comment:
Rather than using `common`, I would suggest naming the file according to what it contains. e.g.

"question-answering-document-identifier.ts" (or id)